messaging/artemis-jta/src/main/java/io/quarkus/ts/openshift/messaging/artemisjta/ProducerService.java

```diff
@@ -30,19 +30,21 @@ public void produceCustomPrice(String customPrice, boolean fail) {
                 throw new IllegalStateException("Bad hair day");
             }
             p.send(context.createQueue("custom-prices-2"), customPrice);
-            LOG.info(customPrice + " sent to queue custom-prices-2");
             context.commit();
+            LOG.info(customPrice + " sent to queue custom-prices-2");
         }
     }
 
     public void produceCustomPriceNoJTA(String customPrice, boolean fail) {
         try (JMSContext context = connectionFactory.createContext(Session.AUTO_ACKNOWLEDGE)) {
             context.createProducer().send(context.createQueue("custom-prices-1"), customPrice);
```

Comment:
@Sgitario `produceCustomPriceNoJTA` shouldn't have `context.commit();` as it uses Session.AUTO_ACKNOWLEDGE

Please remove it

Reply:
Removed.